frontend/src/views/Gallery/Base.vue

```diff
@@ -280,13 +281,13 @@ onBeforeRouteUpdate(async (to, _) => {
         <template v-slot:activator="{ props }">
           <v-fab-transition>
             <v-btn
-              v-show="romsStore.selected.length > 0"
+              v-show="romsStore._selectedIDs.length > 0"
```

Comment:
This has to use the internal list as it ties into the v-model of the data-table

Reply:
Fixed the data-table  sync issue with the grid view changing the item-value of the data-table to the rom `id`